lib/ram_pwrdn/Kconfig

```
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config RAM_POWER_DOWN_LIBRARY
+	bool "Enable API for turning off unused RAM segments"
+	help
+	  This allows application to call API for disabling unused RAM segments.
+	  This is usually not needed, but can improve battery lifetime for
+	  applications that spend most of the time in the sleep mode with most
+	  peripherals disabled.
+
+# Configure RAM_POWERDOWN_LOG_LEVEL
+module =  RAM_POWERDOWN
+module-str = RAM Power down module
+source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
```

Please use new syntax:
```suggestion
source "$(ZEPHYR_BASE)/subsys/logging/Kconfig.template.log_config"
```

From: https://pypi.org/project/kconfiglib/#library-documentation
```
environment variables can be referenced both as $(FOO) (the new syntax) 
```

---

Fixed.